arbitrum-docs/stylus/how-tos/local-stylus-dev-node.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
We can remove this file and add a redirection in the menu, like what we do for running an orbit full node ([see the menu entry here](https://github.com/OffchainLabs/arbitrum-docs/blob/master/website/sidebars.js#L536)). 

We should also add a redirection in vercel, in [this file](https://github.com/OffchainLabs/arbitrum-docs/blob/master/vercel.json#L398-L402).

We can add the current text of this page in a subsection of the main run a local dev node page.

Reply:
@TucksonDev Would you be able to help me with this?
`We should also add a redirection in vercel, in [this file](https://github.com/OffchainLabs/arbitrum-docs/blob/master/vercel.json#L398-L402). Ping me if you need help with this file.`